convert2rhel/unit_tests/utils_test.py

```diff
@@ -460,3 +460,27 @@ def test_remove_orphan_folders(path_exists, list_dir, expected, tmpdir, monkeypa
 
     utils.remove_orphan_folders()
     assert os_remove_mock.call_count == expected
+
+
+class DummyPopenOutput(unit_tests.MockFunction):
+    def __init__(self, output):
+        self.output = output
+
+    def __call__(self, args, stdout, stderr, bufsize):
+        return self
+
+    @property
+    def stdout(self):
+        return self
+
+    def readline(self):
+        return self.output.split()
+
+
+def test_run_subprocess_env(monkeypatch):
+    test_output = DummyPopenOutput("test of nonascii output: cafe".encode("utf-8"))
+    monkeypatch.setattr(utils.subprocess, "Popen", test_output)
+
+    output = utils.run_subprocess(["echo", "foobar"])
+
+    assert "test of nonascii output: cafe" == output
```

Comment:
The `run_subprocess()` function returns a tuple `output, return_code`, not just `output`.

Reply:
I have changed this in my new draft of the unit test.